modules/tinymce/CHANGELOG.md

```diff
@@ -9,13 +9,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - New `expand` function added to `tinymce.selection` which expands the selection around the nearest word. #TINY-9001
 - New `expand` function added to `tinymce.dom.RangeUtils` to return a new range expanded around the nearest word. #TINY-9001
-- New `color_map_background` and `color_map_foreground` option which sets the base colors used in the `backcolor` and `forecolor` toolbar buttons and menu items. #TINY-9184
+- New `color_map_background` and `color_map_foreground` options which set the base colors used in the `backcolor` and `forecolor` toolbar buttons and menu items. #TINY-9184
 - Added optional `storageKey` property to `colorinput` component and `colorswatch` fancy menu item. #TINY-9184
 - New `addView` function added to `editor.ui.registry` which makes it possible to register custom editor views. #TINY-9210
 - New `ToggleView` command which makes it possible to hide or show registered custom views. #TINY-9210
 - New `color_default_foreground` and `color_default_background` options to set the initial default color for the `forecolor` and `backcolor` toolbar buttons and menu items. #TINY-9183
 - New `getTransparentElements` function added to `tinymce.html.Schema` to return a map object of transparent HTML elements. #TINY-9172
-- Added `ToggleToolbarDrawer` event to subscribe to toolbar's opening and closing. #TINY-9271
+- Added `ToggleToolbarDrawer` event to subscribe to toolbar’s opening and closing. #TINY-9271
```

Comment:
We actually want smart quotes? But they cause so many problems ...

Reply:
> We actually want smart quotes? But they cause so many problems

They also visually distinguish themselves from code-centric quote marks. 'value' is meaningfully different from ‘value’.

Reply:
Oh they don't cause real problems. They cause "I've copied and pasted this from a docs page into an input field and it doesn't work. A recent case was I copied and pasted something into a query language input and it didn't interpret the smart quotes properly.

Reply:
6.3 literally has a feature to convert quotes to smart quotes. I'm not sure where I land on this argument, but we have many non-smart quotes in this changelog already.

Reply:
Sounds more like a bug to me :)

Reply:
> but we have many non-smart quotes in this changelog already.

A DOCs ticket in the making, perhaps?